bulk-revoke-tokens-script/Bulk-Revoke-Tokens.ps1

```diff
@@ -0,0 +1,59 @@
+param (
+  [Parameter(Mandatory=$true)][string]$CsvPath,
+  [Parameter(Mandatory=$true)][string]$BearerToken
+)
+
+function RequestDeletion {
+  param (
+    [string]$Json,
+    [string]$BearerToken,
+    [string]$Api
+  )
+  $Headers = @{Authorization=("Bearer {0}" -f $BearerToken)}
+  try
+  {
+    $Response = Invoke-RestMethod -Uri $Api -Method POST -ContentType "application/json" -Body $Json -Headers $Headers
+    @{StatusCode=$Response.BaseRequest.StatusCode; Content=$Response.Content} | Format-Table -AutoSize
+  } catch {
+    @{StatusCode=[int]$_.Exception.Response.StatusCode; Content=$_.ErrorDetails.Message} | Format-Table -AutoSize
+  }
+}
+
+$BatchSize = 100
+$UserIdColName = "user_id"
+
+$CsvObj = Import-Csv -Path $CsvPath
+
+if ($CsvObj | Get-Member -Name $UserIdColName -MemberType NoteProperty)
+{
+  $UserIds = $CsvObj.$UserIdColName
+  if ($UserIds.Count -gt 0)
+  {
+    $Api = "https://www.yammer.com/api/v1/tokens/deletion"
+    $Batch = New-Object Collections.Generic.List[long]
+
+    For ($i = 0; $i -lt $UserIds.Count; $i++)
+    {
+      $Batch.Add([long]($UserIds[$i]))
+
+      if ($Batch.Count -ge $BatchSize)
+      {
+        $Json = ('{{"user_ids": [{0}]}}' -f ($Batch -join ","))
+        RequestDeletion -Json $Json -BearerToken $BearerToken -Api $Api
+        $Batch.Clear()
+      }
+    }
+
+    if ($Batch.Count -gt 0)
+    {
+      $Json = ('{{"user_ids": [{0}]}}' -f ($Batch -join ","))
+      RequestDeletion -Json $Json -BearerToken $BearerToken -Api $Api
+      $Batch.Clear()
+    }
+  }
+}
+else {
```

Comment:
The opening bracket starts on the line after every other conditional statement in this script, so I think we should move it down a line here.